rllib/algorithms/algorithm.py

```diff
@@ -2481,9 +2481,9 @@ def default_resource_request(
     ) -> Union[Resources, PlacementGroupFactory]:
         # Default logic for RLlib Algorithms:
         # Create one bundle per individual worker (local or remote).
-        # Use `num_cpus_for_local_worker` and `num_gpus` for the local worker and
-        # `num_cpus_per_worker` and `num_gpus_per_worker` for the remote
-        # workers to determine their CPU/GPU resource needs.
+        # Use `num_cpus_for_main_process` and `num_gpus` for the local worker and
```

Comment:
Users might ask, what if I run the learner on the main process - do I need `num_gpus`?

Reply:
Good question, I haven't really thought about this. We should definitely retire this option. It's misleading.
If you want PPO for example to learn on the local Learner worker (num_learners=0) AND use the GPU, I guess, we should test, whether this already works with `num_gpus_per_learner=1`. This should, imo, allocate the 1 GPU together with the `num_cpus_for_main_process` CPUs in the placement group bundle. Let me check ...